templates/desktop/tenders/contact-us.html

```diff
@@ -1,146 +0,0 @@
-{% extends "desktop/base_desktop.html" %}
```

Comment:
Isn't this page needed? It's linked to from <http://127.0.0.1:8001/desktop/partners> (although that page should be using a relative rather than absolute URL).